be/src/util/obj_lru_cache.cpp

```diff
@@ -45,4 +45,11 @@ void ObjLRUCache::erase(const ObjKey& key) {
     }
 }
 
+bool ObjLRUCache::exceed_prune_limit() {
+    // just return true to prune all cached obj.
+    // Because ObjLRUCache is counted with number, not memory.
+    // Simple prune all
+    return true;
```

Comment:
great, in addition to obj lru cache, other number lru cache should also have similar override.

Reply:
OK